vars/stageStatusFilename.groovy

```diff
@@ -0,0 +1,19 @@
+// vars/stageStatusFilename.groovy
+
+import java.net.URLEncoder
+
+@NonCPS
+String urlEncode() {
+    return URLEncoder.encode(env.STAGE_NAME, 'UTF-8')
+}
+
+  /**
+   * stageStatusFilename step method
+   *
+   * @param config Map of parameters passed, Unused
+   * returns: String with Url Encoded filename for stage name status.
+   */
+
+String call(Map config= [:]) {
+    return urlEncode() + ".status"
+}
```

Comment:
Might as well add a newline here if you have to push again.

Reply:
Will be in the next push.